packages/amplify-migration/src/command-handlers.ts

```diff
@@ -121,8 +121,7 @@ const getUsageDataMetric = async (): Promise<IUsageData> => {
 const getAccountId = async (): Promise<string | undefined> => {
   const stsClient = new STSClient();
   const callerIdentityResult = await stsClient.send(new GetCallerIdentityCommand());
```

Comment:
Are we validating the `accountId`, in cases where the `stsClient.send(new GetCallerIdentityCommand())` fails for some reason? Is it guaranteed that a valid `accountId` will be fetched here?

Reply:
great question! It's asserted in the consumer `executeStackRefactor` (line 250): 

 `assert(accountId);`

Reply:
Oh yes, got it. Thanks for clarifying!